pkg/webui/components/map/map.styl

```diff
@@ -20,6 +20,9 @@
 .widget
   height: 25rem
 
+.click
+  cursor: pointer !important
```

Comment:
Any way to get this to work without `!important`? (Though don't invest more than 5 minutes in that)